src/redux/data.ts

```diff
@@ -514,19 +1033,36 @@ export const transactionsRemoved = message => async (dispatch, getState) =>
 
     dispatch({
       payload: updatedTransactions,
-      type: DATA_UPDATE_TRANSACTIONS,
+      type: DATA_LOAD_TRANSACTIONS_SUCCESS,
     });
-    dispatch(checkForRemovedNonce(removedTransactions));
+    dispatch(checkForRemovedNonce(removedTransactions as any));
```

Comment:
In this case `removedTransactions` is an array of `RainbowTransaction`s, but `checkForRemovedNonce` wants a `ZerionTransaction`. However, `RainbowTransaction` doesn't have the `address_from` field used in `checkForRemovedNonce`.

I wasn't actually able to simulate a transaction removal event, but my guess is that this does not work as intended. I did print out `transactions` when running the app and it conforms to `RainbowTransaction` as expected (without the `address_from` field that a Zerion transaction would need). Let me know if I'm missing anything though!

Reply:
This context would be good in a comment.